Index: src/CMakeLists.txt
===================================================================
--- src/CMakeLists.txt	(revision 21656)
+++ src/CMakeLists.txt	(working copy)
@@ -141,15 +141,15 @@
 check_include_files (stdlib.h STDC_HEADERS)
 check_include_files (string.h STDC_HEADERS)
 
-set(WORDS_BIGENDIAN)
-if (UNIX)
-  include (TestBigEndian)
-  test_big_endian(WORDS_BIGENDIAN)
-else()
+# set(WORDS_BIGENDIAN)
+# if (UNIX)
+#   include (TestBigEndian)
+#   test_big_endian(WORDS_BIGENDIAN)
+# else()
   # test_big_endian seems to fail with nmake (VS 2010) on windows
   # however, windows is always little endian, so catch this here
   set(WORDS_BIGENDIAN FALSE)
-endif()
+# endif()
 
 # for whatever reason some of the values are not correctly picked up
 # so we encode these values here
